en/mozillavpn.xliff

```diff
@@ -0,0 +1,556 @@
+    </trans-unit>
+    <trans-unit id="vpn.toggle.off">
+      <source xml:space="preserve">Turn VPN off</source>
+      <context-group purpose="location"><context context-type="linenumber">84</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.toggle.isOn">
+      <source xml:space="preserve">VPN is on</source>
+      <context-group purpose="location"><context context-type="linenumber">86</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/main.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.main.productName">
+      <source xml:space="preserve">Mozilla VPN</source>
+      <context-group purpose="location"><context context-type="linenumber">28</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/settings/ViewAboutUs.qml</context><context context-type="linenumber">56</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewInitialize.qml</context><context context-type="linenumber">31</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/settings/ViewAboutUs.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.aboutUs.tos">
+      <source xml:space="preserve">Terms of Service</source>
+      <context-group purpose="location"><context context-type="linenumber">20</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.aboutUs.privacyPolicy">
+      <source xml:space="preserve">Privacy Policy</source>
+      <context-group purpose="location"><context context-type="linenumber">26</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.settings.aboutUs">
+      <source xml:space="preserve">About us</source>
+      <context-group purpose="location"><context context-type="linenumber">36</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/settings/ViewSettingsMenu.qml</context><context context-type="linenumber">49</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.main.productDescription">
+      <source xml:space="preserve">A fast, secure and easy to use VPN. Built by the makers of Firefox.</source>
+      <context-group purpose="location"><context context-type="linenumber">62</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewInitialize.qml</context><context context-type="linenumber">33</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/main.cpp</context><context context-type="linenumber">60</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.aboutUs.releaseVersion">
+      <source xml:space="preserve">Release Version</source>
```

Comment:
Can you give more context about this string? What's the meaning, how is it used?